examples/experimental/native/flexible_primitives_mnist_pytorch/data.py

```diff
@@ -0,0 +1,47 @@
+import logging
```

Comment:
It's just to use Native API to submit a trial.

Reply:
I would prefer that we expose this example not using `native`, since we are expecting this API to be out of `experimental` before `native`.  But I don't feel super strongly here.

Reply:
done